WordPress/src/main/java/org/wordpress/android/ui/blaze/ui/blazeoverlay/BlazeOverlayFragment.kt

```diff
@@ -205,6 +205,18 @@ class BlazeOverlayFragment : Fragment() {
         }
     }
 
+    @Composable
+    fun ScrollableBox(uiModel: BlazeUIModel?,
+                      isDarkTheme: Boolean) {
+        Box(Modifier.fillMaxSize()) {
+            LazyColumn(Modifier.fillMaxSize()) {
+                items(1) {
+                    BlazeOverlayContent(uiModel, isDarkTheme)
+                }
+            }
+        }
+    }
```

Comment:
❤️ @thomashorta 
Funny, I did start out with the following on the column ... but then I also added another LazyColumn
`modifier = Modifier
        .fillMaxSize()
        .verticalScroll(rememberScrollState())`

And that I ran into a bunch of `IllegalStateException`s. I should have realized that after I removed the `LazyColumn` for the bulleted list that it would have been fine. Thank you.

> You will notice that I changed the padding to be applied at all sides now (instead of just top, start, and end). That's because if the content ever fills the entire screen and needs scrolling there will still be a bit of padding below the CTA button.

Excellent 🙇 I was unsure if I needed to add something to preserve the space.

> Side note: LazyColumn is the Compose-equivalent of RecyclerView, so it's better to be used only when there's a high number of items, so only what is visible is actually processed.

That's what I thought too. What really threw my off was that everything was inside a single column, but the list scrolled itself, which is not what we wanted in this case. If we had treated each element as a separate row, then just using a lazyColumn would have been okay. (If I think in terms of a recyclerview).

I'll make the suggested changes now. :)

Reply:
@thomashorta Requested changes in commit [f2da160](https://github.com/wordpress-mobile/WordPress-Android/pull/18083/commits/f2da1606237ce75a494de7ea4e7c972ebe1794a4)

Thanks again for the assistance and the :TIL opportunities. :)